data/human/human missions.txt

```diff
@@ -6849,6 +6849,133 @@ mission "FW Refugees to Humanika"
 		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`
 
+
+mission "Derelict Informer"
```

Comment:
Honestly not so sure on including this mission. It feels very game-y, like it's only purpose is to inform the player about a game mechanic rather than it being an organic encounter. Why not leave derelict ships as something for the player to discover on their own instead of having someone tell you about them?

Reply:
Initially I planned on having the derelicts only in the 2 jump uninhabited systems and would have been difficult to find since these systems are pretty much avoided once discovered. I've since included them in the 1 jump systems which are commonly passed through, so perhaps the mission could be excluded, or expanded to be a proper paying mission while still teaching about derelicts.

Reply:
I think I'd prefer to just leave this mission out.